### PR TITLE
dns/aws: include operator version in UserAgent for AWS API calls

### PR DESCRIPTION
**Pull request by abhinavdahiya**
As example of this terraform-provider-aws [1]

[1]: https://github.com/terraform-providers/terraform-provider-aws/blob/98f11aecb078d82fd7d9b59e5307874797d8f359/aws/config.go#L420

### CODE REVIEW

File: pkg/dns/aws/dns.go [hunk 1]
```diff
@@ -78,6 +80,10 @@ func NewManager(config Config) (*Manager, error) {
 	if err != nil {
 		return nil, fmt.Errorf("couldn't create AWS client session: %v", err)
 	}
+	sess.Handlers.Build.PushBackNamed(request.NamedHandler{
+		Name: "openshift.io/cluster-ingress-operator",
```
Comment (ironcladlou) [inline pkg/dns/aws/dns.go]: can you make this `openshift.io/ingress-operator`

File: pkg/dns/aws/dns.go [hunk 2]
```diff
@@ -78,6 +80,10 @@ func NewManager(config Config) (*Manager, error) {
...
+		Fn:   request.MakeAddToUserAgentHandler("openshift.io cluster-ingress-operator", version.OperatorVersion),
```
Comment (ironcladlou) [inline pkg/dns/aws/dns.go]: "openshift.io ingress-operator" please

File: pkg/dns/aws/dns.go (hunk 1 above)
Comment (abhinavdahiya) [reply pkg/dns/aws/dns.go]: done with https://github.com/openshift/cluster-ingress-operator/compare/a60a53d976a6f8b9312678fdd5582b4716373a5c..b8dfa91aa6707cd3a3af790e4e6524821973034d

File: pkg/dns/aws/dns.go (hunk 2 above)
Comment (abhinavdahiya) [reply pkg/dns/aws/dns.go]: done with https://github.com/openshift/cluster-ingress-operator/compare/a60a53d976a6f8b9312678fdd5582b4716373a5c..b8dfa91aa6707cd3a3af790e4e6524821973034d

**Comment by ironcladlou** [comment]
Thanks!
/lgtm
/retest

**Pull request merged**